src/app/pages/properties/add-property/add-property.component.ts

```diff
@@ -0,0 +1,136 @@
+import { Component, OnInit } from "@angular/core";
+import {
+  FormGroup,
+  FormControl,
+  FormBuilder,
+  Validators
+} from "@angular/forms";
+import { PropertyService } from "src/app/services/property/property.service";
+
+@Component({
+  selector: "app-add-property",
+  templateUrl: "./add-property.component.html",
+  styleUrls: ["./add-property.component.scss"]
+})
+export class AddPropertyComponent implements OnInit {
+  addPropertyForm: FormGroup;
+
+  constructor(
+    private fb: FormBuilder,
+    private propertyService: PropertyService
+  ) {}
+
+  numberLengthLess(control: FormControl): { [key: string]: boolean } | null {
```

Comment:
This custom validation will have to be refactored to cater for arbitrary input, and not hard-coded as it has been. And why did you opt to create the custom validator instead?
@neelxie